README.md

```diff
@@ -1,2 +1,10 @@
 # lean4-nightly
-Nightly builds
+
+This repository hosts automatically generated Lean [toolchains](https://github.com/leanprover/elan#elan-lean-version-manager) for Lean nightly builds.
+
+You should not need to use this repository directly. Instead:
+* Once your Lean pull request has the [`toolchain-available`](https://github.com/leanprover/lean4/pulls?q=is%3Apr+is%3Aopen+label%3Atoolchain-available) label,
+* Edit the `lean-toolchain` file in the project you would like to test your pull request in to say:
+  `leanprover/lean4:nightly-YYYY-MM-DD`, where `YYYY-MM-DD` is the date of the nighly you want to use (e.g. `2023-11-01`).
```

Comment:
No, this isn't right. The `toolchain-available` label is referring to the availability of the `lean-pr-testing-NNNN` toolchain over at the `lean4-pr-releases` repository, not about the nightlies.

Reply:
I think to see if a PR has made it into a nightly one needs to look at the release notes on this repository.

Reply:
Hmm, I don't remember how these sentences ended up in this PR...